chimia.csl

```diff
@@ -0,0 +1,320 @@
+            <date-part name="year"/>
+          </date>
+        </if>
+      </choose>
+    </group>
+  </macro>
+  <!-- END MACROS -->
+  
+  <!-- CITATION -->
+  <citation collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout vertical-align="sup" delimiter="," prefix="[" suffix="]">
+      <text variable="citation-number"/>
+    </layout>
+  </citation>
+  
+  <!-- BIBLIOGRAPHY -->
+  <bibliography entry-spacing="0" second-field-align="flush">
+    <layout suffix=".">
+      <text variable="citation-number" prefix="[" suffix="] "/>
+      <choose>
+        <!-- journal/review article -->
+        <if type="article-journal review" match="any">
+          <text macro="author" suffix=", "/>
+          <group delimiter=" ">
+            <text variable="container-title" form="short" font-style="italic" strip-periods="false"/>
+            <group delimiter=", ">
+              <text macro="year-date"/>
+              <group>
+                <text variable="volume" font-style="italic"/>
+              </group>
+              <text macro="pages"/>
+            </group>
+          </group>
+          <text macro="doi" prefix=", "/>
+        </if>
+        <!-- book and report -->
+        <else-if type="book report bill graphic legal_case legislation motion_picture song" match="any">
+          <text macro="author" suffix=", "/>
+          <group delimiter=", ">
+            <text variable="title" prefix="‘" suffix="’" />
+            <text macro="publisher"/>
+            <text macro="year-date"/>
+          </group>
+          <text macro="doi" prefix=", "/>
+        </else-if>
+        <!-- book chapter or conference proceedings -->
+        <else-if type="chapter paper-conference entry-encyclopedia" match="any">
+          <text macro="author" suffix=", "/>
+          <group delimiter=", ">
+            <group delimiter=" ">
+              <text term="in"/>
+              <text variable="container-title" text-case="title" prefix="‘" suffix="’" />
+            </group>
+            <text macro="volume" font-style="italic"/>
+            <text macro="edition"/>
+            <text macro="editor"/>
+            <text macro="publisher"/>
+            <text macro="year-date"/>
+            <text macro="pages"/>
+          </group>
+          <text macro="doi" prefix=", "/>
+        </else-if>
+        <!-- thesis -->
+        <else-if type="thesis">
+          <text macro="author" suffix=", "/>
+          <group delimiter=", ">
+            <group delimiter=" ">
+              <text macro="thesis-type"/>
+              <text variable="publisher"/>
+              <text macro="thesis-number"/>
+            </group>
+            <text variable="publisher-place"/>
+            <text macro="year-date"/>
+          </group>
+          <text macro="doi" prefix=", "/>
+        </else-if>
+        <!-- computer program (is mapped to type "article" in Mendeley) -->
+        <else-if type="article">
```

Comment:
I leave the "article" part as it is. For Zotero, although rarely used by chemists - they prefer Endnote or Mendeley - I will add a check on version in the "book" section. Although I'm not happy with this. The easiest solution would be to add correct document types, so that reference management software not has to map arbitrarily.